CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # salsify_rubocop
 
+## v0.46.0
+- Update to `rubocop` v0.46.0.
+- Disable new cops: `Bundler/OrderedGems` and `Style/EmptyMethod`.
```

Comment:
How does the ordered gems cop work? I generally like how we use alphabetical order in the dandelion with Gemfile.